index.js

```diff
@@ -1,14 +1,15 @@
 const validateLineup = (lineup) => {
-    
-    //returns false when the lineup has a total salary greater than 45000'
-    
-    const totalSalary = (lineup) => {lineup.map(lineup.salary).reduce((accm, val) => accm  + val, 0)
+    let positionCounts = getPositionCount(lineup) && 
+    let positionCount =  fewPositionCount(lineup) &&
```

Comment:
you have made new variables but never used any of them.  That is what the squiggy lines will tell you when you hover over them or of your text is faded it is not being used.  Use your ESLint